### ISSUE TITLE
Deleting Multiple lines from profile editor causes DataGridView exception error in MSFS

### ISSUE THREAD

**Issue by SoapyGit** [MobiFlight/MobiFlight-Connector#1883]
**Describe the bug**
When editing a profile and deleting multiple lines an error appears
**Steps To Reproduce**
1. Go to a .mcc profile with MFSF 2020 running
2. Select mutiple lines for deletion.  Including the last line in the deletion causes error more reliably. 
3. Error shown in attachement
4. It is not possible to cancel the error and MF UI freeezes - no further input possible and have to kill MF using Task Manager.

**Expected behavior**

**Actual behavior**
A clear and concise description of what you actually happens.

**Details**
 - MobiFlight Version: [e.g. 8.2.0]
 - FlightSim: [e.g. MSFS2020]
 - AddOn: [e.g. Default Aircraft]

**Additional context**
![image](https://github.com/user-attachments/assets/397d1178-afb5-4888-b8df-4d62ae9b6013)

**Comment by neilenns** [comment]
I made several attempts to reproduce this and couldn't. I tried both input and output configs, selecting the last (not real) entry, not selecting the last entry, having the sim running, having MobiFlight in run mode, etc. and it never happened :(

**Comment by SoapyGit** [comment]
From MF Discord post.Thanks for investigating Sent from my iPadOn 20 Dec 2024, at 20:38, Neil Enns ***@***.***> wrote:﻿
I made several attempts to reproduce this and couldn't. I tried both input and output configs, selecting the last (not real) entry, not selecting the last entry, having the sim running, having MobiFlight in run mode, etc. and it never happened :(

**Comment by SoapyGit** [comment]
![IMG_0351](https://github.com/user-attachments/assets/4d8503fc-16db-422a-88c2-c07d17428cf9)

**Comment by SoapyGit** [comment]
Not necessarily but I did on the first occasion that I noticed the issue and as you can see from the screenshot, so did Jaime.Sent from my iPadOn 22 Dec 2024, at 19:12, Sebastian M ***@***.***> wrote:﻿
Are you selecting the last empty row too?

**Comment by neilenns** [comment]
> Are you selecting the last empty row too?

Yes. I tried all sorts of variations and couldn't get it to happen.
